tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, beforeEach, document */
+
+'use strict';
+
+var modules = bender.amd.require( 'ckeditor', 'editor', 'promise' );
+
+var content = document.getElementById( content );
```

Comment:
> The point is that we don't care about the element for now, so it can be anything...

Well we actually do care about the argument that comes to the `create` function, cause if it's a string, then we treat it as a selector and evaluate it. So I think the `create` function could "care" more.